### ISSUE TITLE
Problem when press Problem when press

### ISSUE THREAD

**Issue by AbdulazizSobh** [beck24/spam_login_filter#16]
It does not do what is required and this message is displayed Delete and report as spammer

**Comment by rohit1290** [comment]
Try this repo and check if the problem still exists. If yes, please share a screenshot that will help.
https://github.com/rohit1290/spam_login_filter

**Comment by AbdulazizSobh** [comment]
The user has been deleted, but have they been notified to come back

**Comment by rohit1290** [comment]
No, why do you want to notify the spam use to come back. We want them to stay away from the site. right? Once you report and delete, user is deleted and the same is reported to stopforumspam (if you have the correct api key defined)

**Comment by AbdulazizSobh** [comment]
Thank
This plugin is very effective https://github.com/beck24/elgg_recaptcha
It does not work on the latest version 3

**Comment by dome1117** [comment]
https://github.com/albrtangell/SPAMMER/blob/main/main.py
